SetupAppveyor.ps1

```diff
@@ -0,0 +1,15 @@
+# only run if on appveyor, don't want to disable strong name verification elsewhere
```

Comment:
yep, it's only run on appveyor, however I don't want to hack with someones registry if they decided to run locally for some reason or other